concepticondata/conceptlists/Healey-1970-364.tsv

```diff
@@ -0,0 +1,365 @@
+Healey-1970-364-159	159	knee	1371	KNEE
+Healey-1970-364-160	160	knife	1352	KNIFE
+Healey-1970-364-161	161	knife, bush		
+Healey-1970-364-162	162	knuckle	2824	KNUCKLE
+Healey-1970-364-163	163	ladder	1464	LADDER
+Healey-1970-364-164	164	lance		
+Healey-1970-364-165	165	laugh	1355	LAUGH
+Healey-1970-364-166	166	leaf	628	LEAF
+Healey-1970-364-167	167	leaf decorations		
+Healey-1970-364-168	168	leave	1752	LEAVE
+Healey-1970-364-169	169	leech	2273	LEECH
+Healey-1970-364-170	170	leg	1297	LEG
+Healey-1970-364-171	171	lie down	215	LIE DOWN
+Healey-1970-364-172	172	light	630	LIGHT (RADIATION)
+Healey-1970-364-173	173	lip	478	LIP
+Healey-1970-364-174	174	live	2309	LIVE
+Healey-1970-364-175	175	liver	1224	LIVER
+Healey-1970-364-176	176	long	1203	LONG
+Healey-1970-364-177	177	louse	1392	LOUSE
+Healey-1970-364-178	178	love	923	LOVE
+Healey-1970-364-179	179	low	721	LOW
+Healey-1970-364-180	180	lump		
+Healey-1970-364-181	181	make	734	MAKE
+Healey-1970-364-182	182	male	1553	MALE
+Healey-1970-364-183	183	malicious		
+Healey-1970-364-184	184	man	1554	MAN
+Healey-1970-364-185	185	man, young	76	YOUNG MAN
+Healey-1970-364-186	186	marsupial		
+Healey-1970-364-187	187	mat, sleeping	105	MATTRESS
+Healey-1970-364-188	188	milk	635	MILK
+Healey-1970-364-189	189	molar	1049	MOLAR TOOTH
+Healey-1970-364-190	190	moon	1313	MOON
+Healey-1970-364-191	191	morning	1339	MORNING
+Healey-1970-364-192	192	mosquito	1509	MOSQUITO
+Healey-1970-364-193	193	mother	1216	MOTHER
+Healey-1970-364-194	194	mountain	639	MOUNTAIN
+Healey-1970-364-195	195	mouse	1491	MOUSE
+Healey-1970-364-196	196	mouth	674	MOUTH
+Healey-1970-364-197	197	mouth, inner		
+Healey-1970-364-198	198	mouth, outer		
+Healey-1970-364-199	199	mucus, nasal	208	NASAL MUCUS (SNOT)
+Healey-1970-364-200	200	my	2090	MY
+Healey-1970-364-201	201	name	1405	NAME
+Healey-1970-364-202	202	navel	1838	NAVEL
+Healey-1970-364-203	203	neck	1333	NECK
+Healey-1970-364-204	204	nest	539	NEST
+Healey-1970-364-205	205	newborn	1289	BABY
+Healey-1970-364-206	206	night	1233	NIGHT
+Healey-1970-364-207	207	nipple	796	NIPPLE
+Healey-1970-364-208	208	nose	1221	NOSE
+Healey-1970-364-209	209	nostril	1178	NOSTRIL
+Healey-1970-364-210	210	nothing	687	NOTHING
+Healey-1970-364-211	211	offspring	1801	CHILD (DESCENDANT)
+Healey-1970-364-212	212	old (person)	725	OLD PERSON
+Healey-1970-364-213	213	old (thing)	2113	OLD (USED)
+Healey-1970-364-214	214	one	1493	ONE
+Healey-1970-364-215	215	only	1580	ONLY
+Healey-1970-364-216	216	our	780	OUR
+Healey-1970-364-217	217	outside	762	OUTSIDE
+Healey-1970-364-218	218	paddle	234	PADDLE
+Healey-1970-364-219	219	pain, have a	2561	HURT (SENSE PAIN)
+Healey-1970-364-220	220	penis	1222	PENIS
+Healey-1970-364-221	221	person	683	PERSON
+Healey-1970-364-222	222	perspiration	125	SWEAT (SUBSTANCE)
+Healey-1970-364-223	223	pig	1337	PIG
+Healey-1970-364-224	224	plant	1486	PLANT (SOMETHING)
+Healey-1970-364-225	225	play (drum)	111	DRUMMING
+Healey-1970-364-226	226	quickly	3365	QUICKLY
+Healey-1970-364-227	227	rain	2108	RAINING OR RAIN
```

Comment:
Could this be mapped to 658	RAIN (PRECIPITATION)?

Reply:
ok